app/src/test/java/ingsw/group1/findmyphone/location/GeoPositionTest.java

```diff
@@ -173,4 +182,19 @@ public void getDistanceBetweenReturnsAppropriateDistance() {
                 expected[0], actual
         );
     }
+
+
+    @Test
+    public void constructorLocation() {
+        GeoPosition pos = new GeoPosition(testLocation);
+        assertEquals(exampleLatitude, pos.getLatitude());
+        assertEquals(exampleLongitude, pos.getLongitude());
+    }
+
+    @Test
+    public void getLocation_returnsAppropriate() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The JUnit 4 test method name 'getLocation_returnsAppropriate' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=5068369)